<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>714</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>돋움체</family>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Short Punch Trader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>trader_ui_ShortPunch.png</normaloff>trader_ui_ShortPunch.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 48, 48);
color: rgb(230, 230, 230);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="2" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_right">
      <item row="1" column="0">
       <widget class="QGroupBox" name="gb_info_bot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>701</width>
          <height>251</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>【 탐색 정보 】</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="2">
          <widget class="QPlainTextEdit" name="te_info_sellbot">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="te_info_buybot">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="Line" name="line_bot">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="gb_history_trade">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>701</width>
          <height>251</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>【 거래 이력 】</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTableView" name="tv_history_trade">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_left_bottom">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="te_info_run">
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_left_top">
      <item row="0" column="0">
       <widget class="QGroupBox" name="gb_info_time">
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>101</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>101</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>【 시간 정보 】</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <widget class="QLabel" name="lb_info_date">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>23-01-02 (월)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lb_info_time">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>11:22:33</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="gb_info_setting">
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>121</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>121</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>【 설정 정보 】</string>
        </property>
        <widget class="QLabel" name="lb_time_start">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>[ 시작시각 ]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lb_time_end">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>[ 종료시각 ]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lb_cash_max">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>[ 자본금 ]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="gb_info_server">
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>141</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>141</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>【 서버 정보 】</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <widget class="QLabel" name="lb_info_server">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>131</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 170, 127);</string>
         </property>
         <property name="text">
          <string>접속 서버</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lb_info_account">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>[ 계좌번호 ]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lb_info_cash">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>[ D+2 예수금 ]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="gb_info_run">
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>121</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>121</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>【 동작 정보 】</string>
        </property>
        <widget class="QLabel" name="lb_run_mainbot">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>[ 메인봇 ] 동작 대기</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lb_run_buybot">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>[ 매수봇 ] 동작 대기</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lb_run_sellbot">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>[ 매도봇 ] 동작 대기</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_run_buybot">
         <property name="geometry">
          <rect>
           <x>129</x>
           <y>56</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>▶</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_run_sellbot">
         <property name="geometry">
          <rect>
           <x>129</x>
           <y>86</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>▶</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="Line" name="line_main">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
